jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -664,7 +665,8 @@ def vue_show_all_in_list(self, listname):
         self.list_contents = lc
         self.send_state('list_contents')
 
-        self._viewer.plot_spectral_lines()
+        self._viewer.plot_spectral_lines(global_redshift=self._global_redshift
```

Comment:
This conditional also may be removed and it could just be global_redshift. It will either be 0 or whatever the value was updated to in the GUI.

Reply:
and in the subsequent locations where `plot_spectral_line` and `plot_spectral_line`s is called

Reply:
We are redesigning line shifts in the near future to have this more clearly defined, so I think as a bugfix this is probably fine for now, but we probably should check to make sure this doesn't break any known notebooks that may try advanced (non-advertized) functionality for per-line or per-list redshifts.